inbox/sendmail/smtp/postel.py

```diff
@@ -205,9 +204,9 @@ def _smtp_oauth2_try_refresh(self):
     def _try_xoauth2(self):
         auth_string = "user={}\1auth=Bearer {}\1\1".format(
             self.email_address, self.auth_token
-        )
+        ).encode()
```

Comment:
We don't do SMTP sending from sync-engine, but this was invalid because b64encode needs bytes and returns bytes